python/tk_multi_loader/dialog.py

```diff
@@ -352,6 +348,30 @@ def __init__(self, action_manager, parent=None):
         show_details = self._settings_manager.retrieve("show_details", False)
         self._set_details_pane_visiblity(show_details)
 
+        ################################################
+        # set up welcome window
+        highlight_color = self.palette().highlight().color().name()
+        welcome_msg = (
+            "Welcome! Learn more about the Loader App <a "
+            "href='https://help.autodesk.com/view/SGDEV/ENU/?guid=SG_Supervisor_Artist_sa_integrations_sa_integrations_user_guide_html#the-loader' "
+            "style='text-decoration: none;'><font color='%s'>here.</font></a>"
+            % highlight_color
+        )
+
+        self.welcome_widget = QtGui.QMessageBox(
+            QtGui.QMessageBox.Information,
+            "",
+            welcome_msg,
+            QtGui.QMessageBox.Ok,
+        )
+        # force the QMessageBox to be on top of other dialogs.
+        self.welcome_widget.setWindowFlags(
+            self.welcome_widget.windowFlags() | QtCore.Qt.WindowStaysOnTopHint
```

Comment:
Could you add a comment explaining why you do that instead of using the parent parameter, please?

For the record, this is similar to shotgunsoftware/tk-core#899 and shotgunsoftware/tk-core#907.

I regret the choice I made in tk-core. I would revert that and move the QMessageBox creation in the function that needs it (instead of __init__) so I can use the parent parameter and not have to deal with window flags.

Reply:
Ok, great I'll try that.